### DISCUSSION TITLE
Migration to React Hooks

### DISCUSSION THREAD

**Discussion by lfoliveir4** [dooboolab/react-native-audio-recorder-player#349] [Q&A]
The example of this project in classes is amazing. It's just the way I'm using it now. Could we include an example with React Hooks? Or even migrate the example folder to React Hooks?

**Comment by lfoliveir4** [comment]
@hyochan What do you think of this thread?

**Reply by hyochan** [reply]
I think this will be great~! However, I am not currently thinking of rewriting the whole thing myself again at current status.

**Reply by ehuamaniquispe** [reply]
@hyochan I can provide a simple starting using react hooks if there is no one already, just confirm it so I can start working on it.

**Reply by ghost** [reply]
What would be the realistic benefits of migrating to hooks? The current code works with hook and non-hook codebases. What would make this a must-have, possibly cutting a major and forcing thousands of developers down an upgrade path?
